examples/imageviewer/desktopApp/build.gradle.kts

```diff
@@ -28,7 +24,7 @@ compose.desktop {
             packageName = "ImageViewer"
             packageVersion = "1.0.0"
```

Comment:
`examples/imageviewer/shared/src/desktopMain/resources/ic_imageviewer_round.png` is under `resources` instead of `composeResources`.

Is it expected, or it is not possible to have platform-specific resources yet?

Reply:
because the desktop window API requires java resources. it is the same situation as app icon for an android app - system specific thing

Reply:
The window API [accepts](https://github.com/JetBrains/compose-multiplatform/blob/19331a8a02491dd75e2631ba2a301b489b3a620d/examples/imageviewer/shared/src/desktopMain/kotlin/example/imageviewer/view/ImageViewer.desktop.kt#L60) common `Painter`, which is also the type of the resource function `painterResource`.

Reply:
I saw that we moved some icons to `desktop-icons` - they we can't use indeed. But the situation seems different for `ic_imageviewer_round`?

Reply:
we may but don't do. so, I moved it to a platform specific dir